scipy/spatial/tests/test_distance.py

```diff
@@ -1933,7 +1933,13 @@ def test_hamming_unequal_length():
     y = [1, 0, 1, 0]
     # Used to give an AttributeError from ndarray.mean called on bool
     assert_raises(ValueError, whamming, x, y)
+    
 
+def test_hamming_unequal_length_with_w():
```

Comment:
This should fix the linter 😃 
```suggestion
    assert_raises(ValueError, whamming, x, y)


def test_hamming_unequal_length_with_w():
```